packages/webcert/src/feature/certificate/Inputs/UeTextField.test.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import '@testing-library/jest-dom'
+import { render, screen } from '@testing-library/react'
+import * as redux from 'react-redux'
+import userEvent from '@testing-library/user-event'
+import { fakeTextFieldElement } from '@frontend/common'
+import UeTextField from './UeTextField'
+
+const useSelectorSpy = jest.spyOn(redux, 'useSelector')
+const useDispatchSpy = jest.spyOn(redux, 'useDispatch')
+
+useSelectorSpy.mockReturnValue({})
+useDispatchSpy.mockReturnValue(jest.fn())
+
+const mockQuestion = fakeTextFieldElement({ id: '1' })['1']
+
+it('renders component with correct default values', () => {
+  render(<UeTextField question={mockQuestion} disabled={false} />)
+  const input = screen.getByRole('textbox')
+  expect(input).toHaveValue('')
+})
```

Comment:
We need to make sure that the value from `mockQuestion.value.text` is displayed correctly, so we need to add a value to  `mockQuestion` before checking the input value.

Something like this:
```typescript
const mockQuestion = fakeTextFieldElement({ id: '1', { value: { text: 'foo' }} })['1']

it('renders component with correct default values', () => {
  render(<UeTextField question={mockQuestion} disabled={false} />)
  const input = screen.getByRole('textbox')
  expect(input).toHaveValue('foo')
})
```